tests/test_entities.py

```
@@ -305,6 +308,64 @@ async def test_cover_stop(
 
         mock_bleak_client.write_gatt_char.assert_called()
 
+    async def test_cover_open_propagates_command_errors(
+        self,
+        hass: HomeAssistant,
+        mock_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.15.10)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)